### PR TITLE
Tweak header styles to address design review comments

### PR DESCRIPTION
**Pull request by davidpmccormick**
## What is this PR trying to achieve?

Addressing design review comments in #171. All bar the request to have the input caret 2px width (not possible without hacks AFAIK) have been completed.

### CODE REVIEW

File: client/scss/components/_header.scss [hunk 1]
```diff
@@ -6,7 +6,11 @@ $tweakpoints: (
 );
 
 .header {
-  padding-top: 0.8rem;
+  padding-top: 0.1rem;
```
Comment (jamesgorrie) [inline client/scss/components/_header.scss]: What's this magic number?

File: client/scss/components/_header.scss [hunk 2]
```diff
@@ -15,7 +19,12 @@ $tweakpoints: (
 }
 
 .header__inner {
-  padding-bottom: 0.8rem;
+  padding-bottom: 0.4rem;
```
Comment (jamesgorrie) [inline client/scss/components/_header.scss]: More magic?

**Comment by davidpmccormick** [comment]
> Assuming the magic numbers are just tweak values

Yeah, just that. I'm not sure much more light can be shed on them, though. Would adding an e.g. `// Design tweak` comment help?

**Comment by jamesgorrie** [comment]
> Would adding an e.g. // Design tweak comment help?

Not really sure - I suppose if we knew what it was an offset of.

On the point of tweaks, are we sure we not just adding to some browsers, but then setting it incorrectly in others? pixel perfect is not always achievable (alignment however is).

**Pull request merged**